src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/Responses/FileUploadLogResponse.cs

```diff
@@ -2,6 +2,6 @@
 {
     public class FileUploadLogResponse
     {
-        public long FileUploadLogId { get; set; }
+        public long LogId { get; set; }
```

Comment:
and this, but would we ever what a response back? We already know teh Id

Reply:
hi paul, agreed, all update stuff removed